### PR TITLE
data: Langi10

### PR DESCRIPTION
**Pull request by Langi10**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/4875"></a>

### CODE REVIEW

File: data/Langi10.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
+  "name": "Abdulrahman Omboke a.k.a Langi",
+  "type": "personal",
+  "bio": "I am a fullstack developer",
+  "avatar": "https://github.com/Langi10.png",
```
Comment (imbeshat) [inline data/Langi10.json:5]:
```suggestion
```

**Review by imbeshat — CHANGES_REQUESTED** [review]
`avatar` is no longer required

**Review by Hrishi5111998 — DISMISSED** [review]

**Review by himanshu1221 — CHANGES_REQUESTED** [review]
`avatar` has been depriciated please remove it other than that LGTM:+1:

**Review by SaraJaoude — APPROVED** [review]
looks good

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🎉

**Pull request merged**